readreg.py

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env python3
 import minimalmodbus
 import serial
+import struct
+
+#Support functions
+
+def convertToFloat(input_int):
+	#read_long returns a 32-bit int, convert to float...
```

Comment:
This does not seem to be correct. From the datasheet the integer returned should be divided by 100 to get the floating point value.

Reply:
https://www.developpez.net/forums/attachments/p196506d1451307310/systemes/autres-systemes/automation/probleme-com-modbus-pl7-pro/controllerprotocolv2.3.pdf/